src/components/VContentReport/VContentReportForm.vue

```diff
@@ -10,9 +10,9 @@
         tag="p"
       >
         <template #source>
-          <VLink :href="media.url" class="text-pink hover:underline">{{
-            providerName
-          }}</VLink>
+          <VLink :href="media.url" class="text-pink hover:underline">
```

Comment:
In our current settings I think it would, but HTML already ignores insignificant whitespace (I think) :thinking: Unless Vue encodes whitespace literally? There is a [Vue compiler setting](https://vue-loader.vuejs.org/options.html#compileroptions) that causes it to strip all whitespace that I think would be nice to use so that we don't have to do bizarre things like:

```
<div><
  ComponentName
  :prop="asdf"
  >Something</
  ComponentName></div>
```

:upside_down_face: That's just ridiculous and unpleasant to read or write.